cart.js

```diff
@@ -0,0 +1,77 @@
+let cart = [];
+let items = [
+    { id: 1, name: 'orange', price: 4000 }, // VND
+    { id: 2, name: 'banana', price: 3500 }, // VND
+    { id: 3, name: 'apple', price: 8000 }, // VND
+];
+
+
+
+// Add a product to the cart
+const addProductToCart = (id) => {
```

Comment:
CS5.2: em bổ sung IPO cho function này và các functions phía dưới nha
https://oneamo.github.io/docs/high-quality-code-standard-and-pull-request-guidelines/#well-ipo-documented-and-step-by-step-commented